demo/AngryBots/Release/AngryBots.js

```diff
@@ -97,11 +97,18 @@ function integrateWasmJS(Module) {
   };
   info["global.Math"] = global.Math;
   info["env"] = env;
-  var instance;
-  instance = Wasm.instantiateModule(binary, info).exports;
-  mergeMemory(instance.memory.buffer);
+  if (Wasm.experimentalVersion < 0xc) {
```

Comment:
perhaps `if (Wasm.experimentalVersion < 0xc || typeof WebAssembly == "undefined")` would be more robust (in case some engines haven't yet switched over)?